publicHtml/public/index.html

```diff
@@ -1,15 +1,23 @@
-<DOCTYPE html>
+<!DOCTYPE html>
 <html lang="en">
 <head>
 <meta charset="UTF-8">
   <title></title>
 </head>
 <body>
-  <form action="/send" >
-    Name: <input type="TEXT" name="Name"/>
-    Comment: <input type="TEXT" name="Comment"/>
-    Submit: <input type="SUBMIT"/>  
-  </form>
+    Name: <input type="TEXT" id="nameinp"/>
+    Comment: <input type="TEXT" id="commentinp"/>
+    <input type="SUBMIT" id="submit"/>  
+<script>
+  const button = document.getElementById('submit');
+    button.onclick = () => {
+      const name = document.getElementById('nameinp').value;
+      const comment = document.getElementById('commentinp').value;
+      const xhttp = new XMLHttpRequest();
+      xhttp.open('GET', `/XMLHttpRequest?name=${name}&comment=${comment}`);
```

Comment:
"/XMLHttpRequest" this is your url which you are defining, defining it in a easy way will make you understand the program better without confusion.